guides/embedders/huggingface.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Semantic Search with Hugging Face Inference Endpoints - Meilisearch documentation
```

Comment:
Why are we focussing on semantic search specifically for this one, instead of going with hybrid?

Reply:
I've reused the snippet used by similar guides like Cohere, Cloudflare, etc... to keep consistency, we can definitely go with the hybrid instead.